packages/@aws-cdk/aws-ec2-alpha/lib/route.ts

```diff
@@ -71,45 +73,22 @@ export interface InternetGatewayProps {
 /**
  * Properties to define a VPN gateway.
  */
-export interface VPNGatewayProps {
```

Comment:
This is worrying. This is an exported interface and users might use it for some purposes.

Reply:
If you mean with that they might be extending it somewhere for implementation, we don't bind the contract of keeping the interface aligned even in the main lib(checked with core team for one of the route interfaces), for the existing VPNGW props this remains same overall, only breaking change that we specify is the change from VPNGateway to VPNGatewayV2, explained below why we need to do that.